src/memefactory/server/dev.cljs

```diff
@@ -202,6 +199,11 @@
                                           :access-token-secret "PLACEHOLDER"
                                           :just-log-tweet? false}}}
          :web3-events {:events constants/web3-events}})
+       (mount/except [
+                      ;; #'memefactory.server.emailer/emailer
+                      ;; #'memefactory.server.pinner/pinner
+                      #'memefactory.server.twitter-bot/twitter-bot
```

Comment:
This shouldn't be necesary if we put :just-log-tweet? true unless I missed something

Reply:
I'll rebase, check and get back to you

Reply:
Ok, seems to be working i.e. no crashes. I noticed twitter bot logs errors to the js/concole, completely bypassing our logging facilities - I made a small change there so that we get notified in case of problems.